### PR TITLE
Don't use wget in elemental-iso-add-registration

### PR DESCRIPTION
**Pull request by cbosdo**
The rationale behind this change is that wget is not shipped with SLE Micro while curl is.

### CODE REVIEW

**Comment by davidcassany** [comment]
> The rationale behind this change is that wget is not shipped with SLE Micro while curl is.

Thanks! :+1:

**Review by davidcassany — APPROVED** [review]

File: .github/elemental-iso-add-registration [hunk 1]
```diff
@@ -36,7 +36,7 @@ iso_repack() {
 
   case ${ISO_URL} in
     http*)
-      wget -O "${TEMPDIR}/iso/${ISO_FILE}" "${ISO_URL}"
+      curl -o "${TEMPDIR}/iso/${ISO_FILE}" "${ISO_URL}"
```
Comment (fgiudici) [inline .github/elemental-iso-add-registration:39]: `download.opensuse.org/repositories/...` will be redirected to `ftp.gwdg.de/pub/opensuse/repositories/`-
So, you mean the curl -L option here :-) 
```suggestion
      curl -Lo "${TEMPDIR}/iso/${ISO_FILE}" "${ISO_URL}"
```

**Review by fgiudici — CHANGES_REQUESTED** [review]
Thanks for the PR!
One little change required :-)

**Review by fgiudici — APPROVED** [review]
looks good, thanks!

**Comment by ldevulder** [comment]
What I don't like is that `curl` doesn't do retry and we have sporadic download issue with opensuse.org regularly...

**Pull request merged**